contributing/getting-started-e2e.md

```diff
@@ -455,9 +455,11 @@ On the "Open a pull request" page, you will see dropdowns at the top indicating
 amproject/amphtml / master … <username>/amphtml / <branch name>
 ```
 
-Below this are text boxes where you can provide a title and description for your pull request.  Please follow the guidelines in the template for providing a good title and description.  Make sure to refer to any Issues that you are fixing (by typing "Issue #" and selecting it from the autocomplete) so that people can see which issue you are fixing and people watching the issue will see that there's a PR for it.
+Below this are text boxes where you can provide a title and description for your pull request.  Please follow the guidelines in the template for providing a good title and description.  Make sure to refer to any GitHub issues that you are fixing (by typing "Issue #" and selecting it from the autocomplete) so that people can see which issue you are fixing and people watching the issue will see that there's a PR for it.
```

Comment:
Github has a nice feature now where if you type "Closes #[Issue Number]" in the description, it will autoclose the issue on merge. We could just tell folks to directly say "Fixes #..." or "Closes #...".

Reply:
Great thing to point out; thanks!